postgis/postgis.sql.in

```diff
@@ -3765,6 +3777,13 @@ CREATE AGGREGATE ST_ClusterWithin (geometry, float8) (
 	FINALFUNC = pgis_geometry_clusterwithin_finalfn
 	);
 
+-- Availability: 2.3
+CREATE AGGREGATE ST_DBSCAN (geometry, float8, int) (
```

Comment:
You've established a pattern of ST_Cluster* as the naming convention, would be good to keep?